### PR TITLE
Semana 1 Sesión 3

### PR DESCRIPTION
**Pull request by jorge-benavides**

### CODE REVIEW

File: curso/semana_1/sesion_3/Activity.playground/Resources/avatarseeds.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[
+    "Simon",
```
Comment (alejandrombWize) [inline curso/semana_1/sesion_3/Activity.playground/Resources/avatarseeds.json]:
```suggestion
"Simon",
```

File: curso/semana_1/sesion_3/Activity.playground/Sources/RandomImageController.swift [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    }
+}
+
+extension RandomImageController {
+    private func setupReloadButton() {
+        let button = UIButton.systemButton(with: .init(systemName: "arrow.clockwise")!,
+                                           target: self,
+                                           action: #selector(reloadImage))
+        view.addSubview(button)
+        button.center = .init(x: view.frame.width/2, y: view.frame.height - 50)
```
Comment (alejandrombWize) [inline curso/semana_1/sesion_3/Activity.playground/Sources/RandomImageController.swift]:
```suggestion
        button.center = .init(x: view.frame.width / 2, y: view.frame.height - 50)
```

File: curso/semana_1/sesion_3/StrongReferences.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        apartment.tenant = person
+        person?.aparment = apartment
+    }
+    print("Departamento rentado")
+}
+
+// unowned reference on closure
+do {
+    class Person: Goodbye {
+        var aparment: Apartment?
```
Comment (alejandrombWize) [inline curso/semana_1/sesion_3/StrongReferences.playground/Contents.swift]:
```suggestion
        var apartment: Apartment?
```

File: curso/semana_1/sesion_3/StrongReferences.playground/Contents.swift [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+        apartment.tenant = person
+        person.aparment = apartment
+    }
+    print("Departamento rentado")
+}
+
+// weak reference on closure
+do {
+    class Person: Goodbye {
+        var aparment: Apartment?
```
Comment (alejandrombWize) [inline curso/semana_1/sesion_3/StrongReferences.playground/Contents.swift]:
```suggestion
        var apartment: Apartment?
```

**Review by alejandrombWize — COMMENTED** [review]
Variable name must be "apartment". Everything else looks very good.

File: curso/semana_1/sesion_3/Activity.playground/Resources/avatarseeds.json (hunk 1 above)
Comment (jorge-benavides) [reply curso/semana_1/sesion_3/Activity.playground/Resources/avatarseeds.json]: ✅

File: curso/semana_1/sesion_3/Activity.playground/Sources/RandomImageController.swift (hunk 2 above)
Comment (jorge-benavides) [reply curso/semana_1/sesion_3/Activity.playground/Sources/RandomImageController.swift]: ✅

File: curso/semana_1/sesion_3/StrongReferences.playground/Contents.swift (hunk 3 above)
Comment (jorge-benavides) [reply curso/semana_1/sesion_3/StrongReferences.playground/Contents.swift]: ✅

File: curso/semana_1/sesion_3/StrongReferences.playground/Contents.swift (hunk 4 above)
Comment (jorge-benavides) [reply curso/semana_1/sesion_3/StrongReferences.playground/Contents.swift]: ✅

**Review by alejandrombWize — APPROVED** [review]

**Pull request merged**